example/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Conditionally adding new sources to the main T8 target as in the original BS looked wrong...
+add_library( T8ex )
```

Comment:
Is it possible to link the example binaries with the shared library in the INSTALL_PREFIX? Alternatively, compile in t8code statically?

Reply:
Hmmm yes, I think this should be easy to do. Do you want both at the same time or is some kind of `T8CODE_ENABLE_DYNAMIC_LINKAGE` flag acceptable ?

Reply:
`8CODE_ENABLE_DYNAMIC_LINKAGE` sounds like a viable option.

Reply:
Done